core/testcontainers/compose/compose.py

```diff
@@ -137,9 +139,9 @@ def get_logs(self) -> tuple[bytes, bytes]:
         stdout, stderr = self._docker_compose.get_logs(self.Service)
         return stdout.encode(), stderr.encode()
 
-    def get_wrapped_container(self) -> "ComposeContainer":
+    def get_wrapped_container(self) -> Container:
         """Get the underlying container object for compatibility."""
-        return self
+        return self._docker_compose._get_docker_client().containers.get(self.ID)
```

Comment:
Dumb question: can `.containers.get(self.ID)` return a value other than `Container`, e.g. None? 

what if the container is not found?